testsuite/unittests/test_mat2_psc_exp.sli

```diff
@@ -85,7 +85,7 @@ mm <<
      /time_in_steps true
    >> SetStatus
 
-/spike_detector Create /sp_det Set
+/spike_recorder Create /sp_det Set
```

Comment:
I am just commenting every time I see `sp_det` now... Sorry about that!

Reply:
And I stop writing "Yep." everytime I see one of your comments... OK?!